src/System.Management.Automation/engine/Modules/ModuleIntrinsics.cs

```diff
@@ -1566,7 +1566,8 @@ private static void SortAndRemoveDuplicates<T>(List<T> input, Func<T, string> ke
                     if (SessionStateUtilities.MatchesAnyWildcardPattern(entry.Key, functionPatterns, false))
                     {
                         sessionState.ExportedFunctions.Add(entry.Value);
-                        string message = StringUtil.Format(Modules.ExportingFunction, entry.Key);
+                        string fileName = Path.GetFileName(sessionState.Module.Path);
+                        string message = StringUtil.Format(Modules.DiscoveringFunction, entry.Key, fileName);
```

Comment:
I believe we still want to keep the original `Exporting function ...` text as if you do:

```powershell
import-module module.psm1
```

Then the message is correct.  It's only if there is a .psd1 that applies filtering where it becomes confusing.

Reply:
Done :)